components/Dialogs/src/MDCAlertController.m

```diff
@@ -212,6 +212,34 @@ - (void)setAccessoryViewNeedsLayout {
       [self.alertView calculatePreferredContentSizeForBounds:CGRectInfinite.size];
 }
 
+- (NSTimeInterval)presentationOpacityAnimationDuration {
+  return ((MDCDialogTransitionController *)self.transitioningDelegate).opacityAnimationDuration;
+}
+
+- (void)setPresentationOpacityAnimationDuration:
+    (NSTimeInterval)presentationOpacityAnimationDuration {
+  ((MDCDialogTransitionController *)self.transitioningDelegate).opacityAnimationDuration =
+      presentationOpacityAnimationDuration;
+}
+
+- (NSTimeInterval)presentationScaleAnimationDuration {
+  return ((MDCDialogTransitionController *)self.transitioningDelegate).scaleAnimationDuration;
+}
+
+- (void)setPresentationScaleAnimationDuration:(NSTimeInterval)presentationScaleAnimationDuration {
+  ((MDCDialogTransitionController *)self.transitioningDelegate).scaleAnimationDuration =
+      presentationScaleAnimationDuration;
+}
+
+- (CGFloat)presentationInitialScaleFactor {
+  return ((MDCDialogTransitionController *)self.transitioningDelegate).dialogInitialScaleFactor;
+}
+
+- (void)setPresentationInitialScaleFactor:(CGFloat)presentationInitialScaleFactor {
+  ((MDCDialogTransitionController *)self.transitioningDelegate).dialogInitialScaleFactor =
+      presentationInitialScaleFactor;
+}
```

Comment:
It might be a good follow-up cleanup to create an internal helper method that returns the casted transitioning delegate for readability purposes.

```objc
- (MDCDialogTransitionController *)transitionController {
  return (MDCDialogTransitionController *)self.transitioningDelegate;
}
```

Reply:
I like it! It's easy enough and feels reasonably within scope, so I just added it to this PR.